### PR TITLE
Cleanup a few configs.

### PR DESCRIPTION
**Pull request by dlorenc**

### CODE REVIEW

File: dockerfile_build/test/BUILD [hunk 1]
```diff
@@ -13,10 +13,3 @@ py_binary(
     name = "config_stripper",
     srcs = ["//dockerfile_build:config_stripper.py"],
 )
-
-dockerfile_build(
```
Comment (aaron-prindle) [inline dockerfile_build/test/BUILD]: This rule is used in //reproducible/ubunut:ubuntu_build currently

Comment (dlorenc) [reply dockerfile_build/test/BUILD]: I think this one actually isn't - there's a duplicate one up in dockerfile_build/BUILD that is. I think it was just a copy/paste error when you split tests.

**Comment by aaron-prindle** [comment]
Ah sorry, you're right. Didn't realize this was test

File: dockerfile_build/test/BUILD (hunk 1 above)
Comment (aaron-prindle) [reply dockerfile_build/test/BUILD]: Ah sorry, didn't realize this was /test

**Pull request merged**